_posts/2016-10-11-your-community-manager.md

```diff
@@ -0,0 +1,53 @@
+---
+name: your-community-manager
+layout: post
+title: "Greetings from Your Community Manager!"
+date: 2016-10-11
+authors:
+  - name: Stefanie Butland
+    url: https://ropensci.org/about/
```

Comment:
you can remove this url bit, cause we look for any staff or leadership team names, and link automatically to the about page, so just have `- name: Stefanie Butland`